docs/supression.rst

```diff
@@ -0,0 +1,77 @@
+.. _supression:
+
+Suppression API
+==============
+
+You can manage Postmark's Suppression lists with a few simple calls:
+
+To view the current suppression list:
+.. code-block:: python
+
+    >>> response = postmark.get_suppressions(stream_id="test")
```

Comment:
I think it would be better to unify suppressions with other sub-sections, so it is `postmark.suppressions.all(stream_id="test")`

Reply:
See an example [here](https://github.com/Stranger6667/postmarker/blob/c5717014b6c837502353a8a5361832cb3bc49d7b/src/postmarker/models/domains.py#L37)

Reply:
Gotcha, I think I switched over to a sub-section.